files/build_templates/buffers_config.j2

```diff
@@ -46,7 +46,11 @@ def
             'torrouter_server'       : '5m',
             'leafrouter_torrouter'   : '40m',
             'spinerouter_leafrouter' : '300m',
+            'upperspinerouter_spinerouter' : '50m',
+            'spinerouter_leafrouter' : '300m',
```

Comment:
Why do you need 'spinerouter_leafrouter' : '300m',  twice?

Reply:
Completed